src/logic/logicTests/calculate.test.js

```diff
@@ -0,0 +1,127 @@
+import calculate from '../calculate.js';
+
```

Comment:
We need to write a unit test for `.` operator condition, when `buttonName === .`.

Reply:
add test for dot operator in calculate.test.js

thank you